code/game/objects/structures/janicart.dm

```diff
@@ -300,11 +292,6 @@
 				buckled_mob.pixel_x = -12
 				buckled_mob.pixel_y = 7
 
-
-/obj/structure/stool/bed/chair/janicart/bullet_act(obj/item/projectile/Proj)
-	if(buckled_mob)
-		buckled_mob.bullet_act(Proj)
-

```

Comment:
Bullet stuff for mobs on a tile with a dense object/turf is already handled in projectile/Bump() so this isn't needed.